client/lib/js/utils/request.js

```diff
@@ -0,0 +1,38 @@
+import { request as baseRequest, sanitize } from 'js-utility-belt/es6';
+
+import ApiUrls from '../constants/api_urls';
+
+
+const DEFAULT_REQUEST_CONFIG = {
+    credentials: 'include',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+    }
+};
+
+/**
+ * Small wrapper around js-utility-belt's request that provides default settings and response
+ * handling
+ */
+export default function request(url, config) {
```

Comment:
I'd shy away from it since it really loses flexibility very quickly if we do things like assuming json responses and stripping null query parameters.